### PR TITLE
fix : issue searching with space not show on some feature complaint

### PR DESCRIPTION
**Pull request by marsellavaleria19**
## Changes
[fix : searching with space on some feature complaint](https://github.com/jabardigitalservice/super-app-cms/commit/5a2a4d7dd1c85ed4b0dc6cd43302724be836f8d7)

## Evidence
title: fix issue searching with space not show on some feature complaint
project: Jabar Super Apps
participants: @marsellavaleria19 @rayfajars @azhargraha @agunghide @Ibwedagama 
screenshot: https://s.digitalservice.id/95dqXO

### CODE REVIEW

File: components/Aduan/index.vue [hunk 1]
```diff
@@ -563,6 +563,7 @@ export default {
       if (value.length > 2 || value.length === 0) {
         this.query.page = 1
         this.query.search = value.length > 2 ? value : null
+        this.query.search = value.trim()
```
Comment (Ibwedagama) [inline components/Aduan/index.vue:566]: this `this.query.search` value assignment happen twice @marsellavaleria19 , and also `trim()` function only work on string.

Is this line necessary? @marsellavaleria19 

```
this.query.search = value.length > 2 ? value : null
```

Comment (Ibwedagama) [reply components/Aduan/index.vue:566]:
```
const a = null
console.log(a.trim())
```

running trim method on something that is not a string will cause an error

Comment (Ibwedagama) [reply components/Aduan/index.vue:566]: I think we just need to trim the string if value.length > 2

```
this.query.search = value.length > 2 ? value.trim() : null
```

**Review by Ibwedagama — APPROVED** [review]

File: components/Aduan/index.vue (hunk 1 above)
Comment (marsellavaleria19) [reply components/Aduan/index.vue]: Oh i see. Thank you for advice @Ibwedagama . I've fixed it, this is my commit : https://github.com/jabardigitalservice/super-app-cms/pull/353/commits/96795537a422c1ea2b9bd26812c4d0d8a7bb098c

**Review by azhargraha — APPROVED** [review]

**Review by agunghide — APPROVED** [review]

**Review by rayfajars — APPROVED** [review]

**Pull request merged**